src/tests/ftest/util/command_utils.py

```diff
@@ -154,7 +154,7 @@ def _run_process(self):
 
         except DaosTestError as error:
             # Command failed or possibly timed out
-            raise CommandFailure from error
+            raise CommandFailure(str(error))
```

Comment:
(pylint-raise-missing-from) Consider explicitly re-raising using the 'from' keyword